web/source/osk/visualKeyboard.ts

```diff
@@ -405,7 +405,7 @@ namespace com.keyman.osk {
       if(unit == '%') {
         return v + unit;
       } else { // unit == 'px'
-        return Math.round(v)+unit;
+        return (Math.round(v*100)/100)+unit;
```

Comment:
What is the purpose of rounding the value multiplied by one hundred, then dividing it again to its original magnitude? Please document.

Reply:
`Math.Round(1.034 * 100)/100` = 1.03. Just rounds to 2 decimal places. :)